src/index.js

```diff
@@ -68,6 +68,23 @@ function runAsync (command, options) {
   })
 }
 
+function runAsyncSpawn (command, args, options) {
+  return new Promise((resolve, reject) => {
+    const spawnOptions = Object.assign({shell: true}, options)
+    const asyncProcess = childProcess.spawn(command, args, spawnOptions)
+    asyncProcess.on('close', code => {
+      if (code !== 0) {
+        reject('failed')
```

Comment:
We could send an error object instead of `failed` string which doesn't provide much information. For example: `reject(new Error('Command failed: ${command}'))`